sql/core/src/test/scala/org/apache/spark/sql/streaming/EventTimeWatermarkSuite.scala

```diff
@@ -765,6 +765,55 @@ class EventTimeWatermarkSuite extends StreamTest with BeforeAndAfter with Matche
     }
   }
 
+  test("SPARK-35815: Support ANSI intervals for delay threshold") {
+    Seq(
+      // Conventional form and some variants
+      (Seq("3 days", "Interval 3 day", "inTerval '3' day"), 259200000),
+      (Seq("5 hours", "INTERVAL 5 hour", "interval '5' hour"), 18000000),
+      (Seq("8 minutes", "interval 8 minute", "interval '8' minute"), 480000),
+      (Seq("10 seconds", "interval 10 second", "interval '10' second"), 10000),
```

Comment:
Could you replace the magic numbers via constants like `10 * MILLIS_PER_SECOND` Now it is hard to check correctness of the expected values. Especially, the values below.